__device-tests__/gutenberg-editor-image.test.js

```diff
@@ -58,43 +60,22 @@ describe( 'Gutenberg Editor Image Block tests', () => {
 			await editorPage.dismissKeyboard();
 			// end workaround
 
-			imageBlock = await editorPage.getImageBlockAtPosition( 1 );
-			await imageBlock.click();
+			imageBlock = await editorPage.getBlockAtPosition( imageBlock );
 			await swipeUp( driver, imageBlock );
 			await editorPage.enterCaptionToSelectedImageBlock( testData.imageCaption );
 			await editorPage.dismissKeyboard();
-			imageBlock = await editorPage.getImageBlockAtPosition( 1 );
-			await imageBlock.click();
 		}
-		await editorPage.removeImageBlockAtPosition( 1 );
-	} );
-
-	it( 'should be able to add an image block with multiple paragraph blocks', async () => {
-		await editorPage.addNewImageBlock();
-		let imageBlock = await editorPage.getImageBlockAtPosition( 1 );
-
-		// Can only add image from media library on iOS
-		if ( ! isAndroid() ) {
-			await editorPage.selectEmptyImageBlock( imageBlock );
-			await editorPage.chooseMediaLibrary();
-
-			imageBlock = await editorPage.getImageBlockAtPosition( 1 );
-			await imageBlock.click();
-			await swipeUp( driver, imageBlock );
-			await editorPage.enterCaptionToSelectedImageBlock( testData.imageCaption );
-			await editorPage.dismissKeyboard();
-		}
-
-		await editorPage.addNewParagraphBlock();
-		const paragraphBlockElement = await editorPage.getParagraphBlockAtPosition( 2 );
```

Comment:
I removed adding image block since we already have one in this test. It will save some time for us :)